test/operations/manager.go

```diff
@@ -551,31 +546,16 @@ func (m *Manager) GetCurrentGlobalExitRootFromSmc(ctx context.Context) (*etherma
 	return &result, nil
 }
 
-// CheckTrustedExitRootSynced waits until the trusted exitroot is synced.
-func (m *Manager) CheckTrustedExitRootSynced(ctx context.Context, depositCnt uint64) error {
-	return operations.Poll(defaultInterval, defaultDeadline, func() (bool, error) {
-		ger, err := m.storage.GetLatestTrustedExitRoot(ctx, nil)
-		if err != nil {
-			return false, err
-		}
-		count, err := m.storage.GetDepositCountByRoot(ctx, ger.ExitRoots[0][:], 0, nil)
-		if err != nil {
-			if err == gerror.ErrStorageNotFound {
-				return false, nil
-			}
-			return false, err
-		}
-		return count >= uint(depositCnt), nil
-	})
-}
-
 // DeployERC20 deploys erc20 smc
 func (m *Manager) DeployERC20(ctx context.Context, name, symbol string, network NetworkSID) (common.Address, *ERC20.ERC20, error) {
 	client := m.clients[network]
 	auth, err := client.GetSigner(ctx, accHexPrivateKeys[network])
 	if err != nil {
 		return common.Address{}, nil, err
 	}
+	if network == L2 {
```

Comment:
Maybe here we should put a TODO REMOVE because we should pay fees for deploying an smc in L2